examples/RunExampleComplexSuiteWithDBMSOutputReporter.sql

```diff
@@ -0,0 +1,45 @@
+--Shows how to create a test suite with the default reporter which is dbms_output
+--No tables are used for this.   
+--Suite Management packages are when developed will make this easier.
+Clear Screen
+Set Serveroutput On Size Unlimited
```

Comment:
I would change the example scripts to make them self contained by:
- calling script to create example ut package(s) before execution of script
```sql
@ut_exampletest.pks
@ut_exampletest.pkb
@ut_exampletest2.pks
@ut_exampletest2.pkb
```
- drop the package after the script is finished (optionally)